features/step_definitions/basic_steps.rb

```diff
@@ -0,0 +1,26 @@
+  When ("I visit the site") do
+    visit root_path
+  end
+  
+  Then ("I should see {string}") do |content|
+    expect(page).to have_content content
+  end
```

Comment:
You should get into the habit of placing your assertions in a separate file

Reply:
Okay, I will create assertions.rb file and move those lines of code accordingly.